tests/plugins/test_bump_release.py

```diff
@@ -215,8 +218,9 @@ def getBuild(self, build_info):
         labels = {
             'com.redhat.component': 'component1',
             'version': 'fc26',
-            'release': base_release
         }
+        if base_release:
```

Comment:
why this if? 

if base_release will be None, it will be set to '0'

Reply:
Without this block the tests would pass `base_release` as a string `'None'`, but not `None`

Reply:
ahh ok